client/scripts/controllers/chain/ethereum/compound/chain.ts

```diff
@@ -47,9 +56,9 @@ export default class CompoundChain extends EthereumChain {
         this.compoundApi.Token,
       );
       const gasLimit = await contract.estimateGas.delegate(address, amount);
+      console.log(this.balanceOf(address));
```

Comment:
Remove, but also adjust:     

const votesByDecimals = votes.div(BigNumber.from(10).pow(BigNumber.from(this.compoundApi.decimals)));